src/components/sendAskMoney/SendAskMoney.vue

```diff
@@ -710,25 +728,41 @@
       },
 
       async sendTransaction(): Promise<void> {
+        if (this.amount <= 0) {
+          this.errors.transfer.amount = "Le montant à transférer doit être un nombre positif"
+          return
+        }
         let recipient = this.$store.state.lokapi.recipient
         try {
           this.isSendingMoney = true
           await recipient.transfer(this.amount.toString(), this.message)
         } catch (err) {
           this.isSendingMoney = false
+          console.log('Payment failed:', err)
           // {RequestFailed, APIRequestFailed, InvalidCredentials, InvalidJson}
           if (err.message === "User canceled the dialog box") {
             this.$toast.warning(`Transaction en cours annulée`, {
-              position: "top-right",
+              position: "top",
             })
-            return
+          } else {
+            this.$toast.error(
+              `Une erreur inattendue est survenue pendant le transfert d'argent. `+
+              `Veuillez nous excuser pour la gêne occasionnée.<br>` +
+              `Vous pouvez réessayer. Si l'erreur persiste, veuillez contacter votre administrateur.`,
+              {
+                position: "top",
+                duration: 8000,
+                dismissable: true
+              });
           }
-          console.log("Payment failed:", err.message)
-          throw err
+          return
         }
+        this.errors.transfer.amount = false
+        this.showModalFrame1 = false
+        this.showModalFrame2 = false
         this.isSendingMoney = false
         this.$toast.success(`Paiement effectué à ${this.recipientName}`, {
```

Comment:
Same change needed here

Reply:
@SeddikKadi Your PR #220 is not validated nor merged yet because we are waiting for a UI change in #216 on your part. Can you address the issue please ?